introduction.md

```diff
@@ -0,0 +1,2 @@
+Bonjour, je m'appelle Mahdi AHMED. Je suis passionn� par la programmation.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.